apps/inkbeard/src/components/GlobalNavigation.vue

```diff
@@ -108,19 +108,20 @@
         @click="isActive = !isActive"
         @keydown="isActive = !isActive"
       />
-      <AppButton
+      <div
```

Comment:
There wasn't a reason to make this an AppButton or a button, so turned it into a basic `div`.